### PR TITLE
Disable Light Client model-based tests by default

### LINKED ISSUE #968: Model-based tests take a long time
**Steps to reproduce**

Running `cargo test`, I get
```
Running tests/model_based.rs (target/debug/deps/model_based-0199679b3102f8af)

running 1 test
test run_model_based_single_step_tests has been running for over 60 seconds
test run_model_based_single_step_tests ... ok
```

after several minutes.  This increases the cycle time of working on the codebase.  It would be better if the model-based tests were off by default, and ran in CI, to keep the test workflow quick-by-default.

**What's the definition of "done" for this issue?**

Default testing workflow is faster.

### PR DESCRIPTION
**Pull request by thanethomson**
Closes #968 

* [x] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [x] Added entry in `.changelog/`

### CODE REVIEW

File: .changelog/config.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+project_url = 'https://github.com/informalsystems/tendermint-rs'
```
Comment (xla) [inline .changelog/config.toml:1]: This change seems unrelated to the premise of this PR, could you contextualise what constitutes this change?

**Review by xla — APPROVED** [review]
Besides the one comment, this is looking good.

🏢 ☺️ 🈴 🐝

File: .changelog/config.toml (hunk 1 above)
Comment (thanethomson) [reply .changelog/config.toml:1]: It's a prerequisite for being able to automatically generate changelog entries from the CLI using unclog, as per https://github.com/informalsystems/unclog#adding-a-new-unreleased-entry

I could've probably made the change in a separate PR, but it's so small, and I had to do it anyways, that I thought I'd just add it in here.
